spec/spec_helper.rb

```diff
@@ -2,7 +2,10 @@
 require 'simplecov-rcov'
 SimpleCov.formatter = SimpleCov::Formatter::RcovFormatter
 SimpleCov.minimum_coverage 100
-
+SimpleCov.start do
+  add_filter "/.bundle/"
```

Comment:
Actually, on reflection, I'll use the example path from the bundler documentation: `vendor/bundle`.